freestyle/shared/src/test/scala/freestyle/implicits.scala

```diff
@@ -0,0 +1,35 @@
+package freestyle
+
+import cats.implicits._
+import cats.Id
+import org.scalatest.{Matchers, WordSpec}
+
+class implicitsTests extends WordSpec with Matchers {
+
+  "Implicits" should {
+
+    import algebras._
+    import freestyle.implicits._
+
+    "make monad implicitly available" in {
+      type G[A] = FreeS[SCtors1.Op, A]
+      implicitly[cats.Monad[G]].isInstanceOf[cats.Monad[G]]
+    }
+
+    "enable sequenceU" in {
+      val s = SCtors1[SCtors1.Op]
+      val program = for {
+        a <- List(s.x(1), s.x(2)).sequenceU
+      } yield a
+      program.isInstanceOf[FreeS[SCtors1.Op, List[Int]]] shouldBe true
```

Comment:
Should we run `program` instead of using `isInstanceOf` ?

Something like 

```scala
import interps._
program.exec[Option] shouldBe (Some(List1, 2))
```